system-variables.md

```diff
@@ -1004,6 +1004,17 @@ mysql> SHOW GLOBAL VARIABLES LIKE 'max_prepared_stmt_count';
 - Default value: (system dependent)
 - This variable shows the system time zone from when TiDB was first bootstrapped. See also [`time_zone`](#time_zone).
 
+### `tidb_accelerate_user_creation_update` <span class="version-mark">New in v9.0.0</span>
+
+- Scope: GLOBAL
+- Persists to cluster: Yes
+- Applies to hint [SET_VAR](/optimizer-hints.md#set_varvar_namevar_value): No
+- Type: Boolean
+- Default value: `OFF`
+- When too many (for example 100K) SQL users are created in the database, operations such as create user/modify password/change user privilege would suffer severe performance impact. The reason is that every modification to the user cause reload of all users' data. Setting the variable to `ON` is best practice is such scenario.
```

Comment:
```suggestion
- When too many (for example 100,000) SQL users are created in the database, subsequent operations such as user creation, password modification, and permission setting might experience significant performance degradation. This is because every user update requires reloading all user data. In this scenario, it is recommended to set this variable to ON.
```